Coffee_loop/src/CoffeeLoop.java

```diff
@@ -0,0 +1,185 @@
+            }
+
+        }
+        if(number.equals("2"))
+        {
+            if(array1[0] < 350)
+            {
+                System.out.println("Sorry, not enough water!");
+            }
+            else if(array1[1] < 75)
+            {
+                System.out.println("Sorry, not enough milk!");
+            }
+            else if(array1[2] < 20)
+            {
+                System.out.println("Sorry, not enough coffee beans!");
+            }
+            else
+            {
+                System.out.println("I have enough resources, making you a coffee!");
+                array1[0] -= 350;
+                array1[1] -= 75;
+                array1[2] -= 20;
+                array1[3] -= 1;
+                array1[4] += 7;
+            }
+
+
+        }
+
+        if(number.equals("3"))
+        {
+            if(array1[0] < 200)
+            {
+                System.out.println("Sorry, not enough water!");
+            }
+            else if(array1[1] < 100)
+            {
+                System.out.println("Sorry, not enough milk!");
+            }
+            else if(array1[2] < 12)
+            {
+                System.out.println("Sorry, not enough coffee beans!");
+            }
+            else
+            {
+                System.out.println("I have enough resources, making you a coffee!");
+                array1[0] -= 200;
+                array1[1] -= 100;
+                array1[2] -= 12;
+                array1[3] -= 1;
+                array1[4] += 6;
+            }
+
+        }
+
+
+        return array1;
+    }
+
+    public static int[]  fill(int[] array1) throws IOException{
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        System.out.println("Write how many ml of water do you want to add:");
+        System.out.print(">");
+        int countOfWater = Integer.parseInt(reader.readLine());
+        System.out.println( countOfWater);
+        array1[0] += countOfWater;
+
+        System.out.println("Write how many ml of milk do you want to add:");
+        System.out.print(">");
+        int countOfMilk = Integer.parseInt(reader.readLine());
+        System.out.println(countOfMilk);
+        array1[1] +=  countOfMilk;
+
+        System.out.println("Write how many grams of coffee beans do you want to add:");
+        System.out.print(">");
+        int countOfCoffeeBeans = Integer.parseInt(reader.readLine());
+        System.out.println(countOfCoffeeBeans);
+        array1[2] += countOfCoffeeBeans;
+
+        System.out.println("Write how many disposable cups of coffee do you want to add:");
+        System.out.print(">");
+        int countOfCups = Integer.parseInt(reader.readLine());
+        System.out.println(countOfCups);
+        array1[3] += countOfCups;
+
+        //  reader.close();
+
+        return array1;
+    }
+
+    public static int[]  take(int[] array1) throws IOException{
+        System.out.println("I gave you " + array1[4] );
```

Comment:
and what going to happen if array[4] will be null ?

Reply:
done